### DISCUSSION TITLE
При локальном запуске библиотека ищет всё что нужно, но при запуске на хостинге нашёлся лишь единый трек.

### DISCUSSION THREAD

**Discussion by tjmaa1ov** [MarshalX/yandex-music-api#580] [Q&A] [answered]
`searcher = client.search(str(query)) `
        `if not searcher.tracks:`
        `    return "notSearched"`
        `if searcher.tracks["results"][0] == {}:`
        `    return "notSearched"`
        `client.tracks(f"{searcher.tracks['results'][0]['id']}")[0].download(f"songs/{ctx.guild.id}.mp3")`
        `payload = {`
        `    "title": searcher.tracks["results"][0]["title"],`
        `    "image": searcher.tracks["results"][0]["og_image"]`
        `}`
        `return payload`
Используется код, предоставленный выше, при локальном запуске этот трек находится:
![image](https://user-images.githubusercontent.com/88947797/208484428-4a690e50-b400-4081-a9d1-4faf39c43205.png)
Но при запуске на хостинге нашлась совсем другой трек:
![image](https://user-images.githubusercontent.com/88947797/208484515-23cc4631-3f16-422b-8d8b-b6a146d6754e.png)
Другие треки на хостинге он вовсе не находит. Как это решить?

**Comment by MarshalX** [comment, marked as answer]
Вероятно потому что хостинг находится не в СНГ. Яндекс Музыка работает без подписки и авторизации только в СНГ
